ar.json

```diff
@@ -163,14 +170,17 @@
       "pageLabel": "صفحة {page}",
       "passwordHide": "اخفاء كلمة المرور",
       "passwordShow": "عرض كلمة المرور",
+      "prevPageLabel": "الصفحة السابقة",
```

Comment:
This is incorrect there is already `previousPageLabel` please report to PrimeNG. See: https://github.com/primefaces/primelocale/pull/170